modules/kraken_client.nf

```diff
@@ -97,21 +97,27 @@ process bracken {
         path "${params.database_set}.bracken_summary.txt", emit: summary
         path "${params.database_set}.bracken_report.txt", emit: report
     """
-    bracken \
-      -d "${database}" \
-      -i "${kraken_report}" \
-      -r "${bracken_length}" \
-      -l "${params.bracken_level}" \
-      -o "${params.database_set}.bracken_summary.txt" \
-      -w "${params.database_set}.bracken_report.txt"
+    if [[ $(wc -l <${kraken_report} -ge 3 ]]
```

Comment:
Yep this looks good, I'll add handling for this in the execution trace parsing so that the error is informative

Reply:
Actually, it would probably be nicer to exit with a known exit code here since if there's nothing identified by kraken we probably want validation to fail at this point since it'll be rejected

Reply:
I think in the CLIMB setup we do want to spot cases where there is nothing identified by kraken. Maybe I should add something optional here though

Reply:
By the looks of it,  it still fails if all the reads are either unclassified or human. So this isn't actually a solution

Reply:
I've undone this change. It didn't solve the problem